Source/WebInspectorUI/UserInterface/Views/Sidebar.js

```diff
@@ -191,6 +208,54 @@ WI.Sidebar = class Sidebar extends WI.View
     {
         // Implemented by subclasses if needed.
     }
+    
+    sizeDidChange()
+    {
+        if (!this.canMoveToBottom)
```

Comment:
`canMoveToBottom` is implicit from `this._side === WI.Sidebar.Sides.Trailing` so id just use that instead

Reply:
`this._side === WI.Sidebar.Sides.Trailing` applies to `SingleSidebar`s as well though, so I've changed `canMoveToBottom` to an internal flag that's false in that subclass and then in the main class check for both.